### PR TITLE
Gitlabsource documentation

### PR DESCRIPTION
**Pull request by tzununbekov**
Documentation for recently introduced [Gitlabsource](https://github.com/knative/eventing-contrib/tree/master/gitlab)

**NOTE:** Gitlabsource has not yet been released, so the installation section looks a bit ugly:
https://github.com/tzununbekov/docs/blob/6b2c0d53e7b102fcafaed15ba63fd169727eef00/docs/eventing/samples/gitlab-source/README.md#install-gitlab-event-source

### CODE REVIEW

File: docs/eventing/samples/gitlab-source/README.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+   [configured with a domain name](../../../serving/using-a-custom-domain.md)
+   that allows GitLab to call into the cluster.
+1. If you're using GKE, you'll also want to
+   [assign a static IP address](../../../serving/gke-assigning-static-ip-address.md).
+1. Install [Knative Eventing](../../../eventing).
+
+### Install GitLab Event Source
+
+GitLab Event source lives in the [knative/eventing-contrib](https://github.com/knative/eventing-contrib). Head to the releases page, find the latest release with `gitlab.yaml`
+artifact and replace the `<RELEASE>` with version tag:
```
Comment (mattmoor) [inline docs/eventing/samples/gitlab-source/README.md]: Remove trailing whitespace:
```suggestion
artifact and replace the `<RELEASE>` with version tag:
```

File: docs/reference/eventing/eventing-contrib.md [hunk 2]
```diff
@@ -2062,31 +2401,55 @@ name to use as the sink.</p>
...
+Source.</p>
 </td>
 </tr>
+</tbody>
+</table>
+<h3 id="sources.knative.dev/v1alpha1.SecretValueFromSource">SecretValueFromSource
+</h3>
+<p>
+(<em>Appears on:</em>
+<a href="#sources.knative.dev/v1alpha1.KafkaSourceSASLSpec">KafkaSourceSASLSpec</a>,
```
Comment (mattmoor) [inline docs/reference/eventing/eventing-contrib.md:2428]: Remove trailing whitespace:
```suggestion
<a href="#sources.knative.dev/v1alpha1.KafkaSourceSASLSpec">KafkaSourceSASLSpec</a>,
```

**Comment by sebgoa** [comment]
@tzununbekov can you fix the conflict ?

**Comment by tzununbekov** [comment]
I removed generated eventing-contrib.md because it had conflicted with one from master which also has some conflicts:
https://github.com/knative/docs/blob/cc6d24a274e8eac8af9373a6b94642093f5931d1/docs/reference/eventing/eventing-contrib.md#kafkasourcetlsspec

**Comment by tzununbekov** [comment]
@googlebot I consent.

**Comment by tzununbekov** [comment]
@googlebot I consent.

**Comment by matzew** [comment]
@tzununbekov  this commit: https://github.com/knative/docs/pull/2353/commits/43df92a9c7eae72fa2304e8d2d06cfff94b623bc 

is committed w/ Matt (well the bot) and you.

can you squash that ? 

that will fix the CLA

**Pull request merged**